Code/ObjectMapping/RKObjectMapping.m

```diff
@@ -69,8 +69,8 @@ - (RKObjectMapping *)invertMapping:(RKObjectMapping *)mapping withPredicate:(BOO
     inverseMapping = [RKObjectMapping requestMapping];
     (self.invertedMappings)[dictionaryKey] = inverseMapping;
     [inverseMapping copyPropertiesFromMapping:mapping];
-    // We want to serialize `nil` values
-    inverseMapping.assignsDefaultValueForMissingAttributes = YES;
+    // We potentially want to serialize `nil` values, adopt the strategy from the mapping we're inverting.
```

Comment:
Arguably. If I create a mapping with assignsDefaultValueForMissingAttributes = NO and I ask for the inverse, I wouldn't expect assignsDefaultValueForMissingAttributes to suddenly be YES. I'd expect the mirror of the mappings and settings.